index.html

```diff
@@ -40,15 +50,16 @@ <h2 class="subtitle">Apache Kafka is a powerful, scalable, fault-tolerant distri
             <li><a href="filter-a-stream-of-events/ksql.html">Filter a stream of events</a></li>
             <li><a href="rekey-a-stream/ksql.html">Rekey a stream with a value</a></li>
             <li><a href="rekey-with-function/ksql.html">Rekey a stream with a function</a></li>
-            <li><a href="changing-serialization-format/ksql.html">Convert a stream's serialization format</a></li>
-            <li><a href="split-a-stream-of-events-into-substreams/ksql.html">Split a stream of events into substreams</a></li>
-            <li><a href="merge-many-streams-into-one-stream/ksql.html">Merge many streams into one stream</a></li>
-            <li><a href="finding-distinct-events/ksql.html">Finding distinct events</a></li>
-            <li><a href="udf/ksql.html">Apply a custom, User-Defined Function</a></li>
-            <li><a href="handling-deserialization-errors/ksql.html">Handling deserialization errors</a></li>
-            <li><a href="dynamic-output-topic/kstreams.html">Choosing the output topic dynamically</a></li>
-            <li><a href="naming-stateful-operations/kstreams.html">Naming stateful operations</a></li>
+            <li class="more is-hidden"><a href="changing-serialization-format/ksql.html">Convert a stream's serialization format</a></li>
+            <li class="more is-hidden"><a href="split-a-stream-of-events-into-substreams/ksql.html">Split a stream of events into substreams</a></li>
+            <li class="more is-hidden"><a href="merge-many-streams-into-one-stream/ksql.html">Merge many streams into one stream</a></li>
+            <li class="more is-hidden"><a href="finding-distinct-events/ksql.html">Finding distinct events</a></li>
+            <li class="more is-hidden"><a href="udf/ksql.html">Apply a custom, User-Defined Function</a></li>
+            <li class="more is-hidden"><a href="handling-deserialization-errors/ksql.html">Handling deserialization errors</a></li>
+            <li class="more is-hidden"><a href="dynamic-output-topic/kstreams.html">Choosing the output topic dynamically</a></li>
+            <li class="more is-hidden"><a href="naming-stateful-operations/kstreams.html">Naming stateful operations</a></li>
           </ul>
+          <div id="show"><a>More</a></div>
```

Comment:
Very cool. For your consideration, adding some classes to the More anchor would visually differentiate the toggle from the other links ...
```suggestion
          <div id="show"><a class="button is-small is-fullwidth">More</a></div>
```

![image](https://user-images.githubusercontent.com/141130/83443996-af926a00-a418-11ea-932a-36e359b4ad6c.png)